wiki/Tournaments/CIC/2018/en.md

```diff
@@ -0,0 +1,373 @@
+---
+tags:
+- CIC 2018
+- CIC2018
+---
+# Catch the Beat Indonesia Cup 2018
+
+![CIC 2018 Logo](img/logo.jpg)
+
+The **Catch the Beat Indonesia Cup 2018** (***CIC 2018***) is Indonesian one-on-one osu!catch tournament hosted by ![][flag_ID] [\[490\]Kazuyo](/users/1830361). It is the 6th installment of the Catch the Beat Indonesia Cup 2018.
+
+## Tournament Schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration Phase | 2018-07-28/2018-08-05 |
+| Group Stage | 2018-08-06/2018-08-16 |
+| Round of 16 | 2018-08-17/2018-08-19 |
+| Quarterfinals | 2018-08-20/2018-08-26 |
+| Semifinals | 2018-08-27/2018-09-02 |
+| Finals | 2018-09-03/2018-09-09 |
+
+## Prizes
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold Crown](/wiki/shared/GCrown.png "1st place") | Profile badge and 50% of the prize pool |
```

Comment:
`Profile badge and 50% of the prize pool` -> `50% of the prize pool, profile badge`